client/src/auth/isAuth.js

```diff
@@ -1,11 +1,8 @@
 const axios = require('axios');
 
 export default function isAuth() {
-  // axios
-  //   .get('/api/v1/isAuthenticated')
-  //   .then(result => (result.data.isAuthenticated ? true : false))
-  //   .catch(e => {
-  //     return false;
-  //   });
-  return true;
+  axios
```

Comment:
@AymanAlqoqa Do you need to return here? i.e. `return axios.get(....)`

Reply:
you are right.... thanks alot :)